models/models.js

```diff
@@ -3,32 +3,55 @@ const db = require("../db/connection.js");
 
 function selectTopics () {
   return db.query(`SELECT * FROM topics;`).then((result) => {
-    return result.rows;
-  });
-};
+    return result.rows
+  })
+}
 
 function selectArticlesByID (article_id) {
   return db.query(`SELECT * FROM articles WHERE article_id=$1;`, [article_id])
   .then(({ rows }) => {
-    const article = rows[0];
+    const article = rows[0]
     if (!article) {
       return Promise.reject({
         status: 404,
         msg: `No article found for article_id: ${article_id}`,
-      });
+      })
     }
-    return article;
-  });
-};
+    return article
+  })
+}
 
 function selectUsers () {
   return db.query(`SELECT * FROM users;`).then((result) => {
-    return result.rows;
-  });
-};
+    return result.rows
+  })
+}
+
+function updateVote (votes, articleId) {
```

Comment:
The things being passed in here (from line 33 of your controller) are not the votes and the articleId - they're the req.body and req.query (which have the inc_votes and the article_id as values within each). This disparity between what these are named and how they are being referenced causes confusing and unnecessary lines of code like that on line 31. If you had destructured the values from each one in the controller before passing this into the function, your code would be far more readable.

Reply:
changed